lib/load-simple-icons.js

```diff
@@ -26,14 +27,16 @@ function loadSimpleIcons() {
     // (e.g. 'Hive'). If a by-title reference we generate for
     // backwards compatibility collides with a proper slug from Simple Icons
     // then do nothing, so that the proper slug will always map to the correct icon.
-    if (!(title in originalSimpleIcons)) {
-      simpleIcons[title] = icon
+    // Starting in v7, the exported object with the full icon set has updated the keys
+    // to include a lowercase `si` prefix, and utilizes proper case naming conventions.
+    if (!(`si${title}` in originalSimpleIcons)) {
```

Comment:
I didn't have enough motivation to compare each pair of the 2k plus keys between the two objects to see whether it's always just a 3rd character capitalization, so I've opted to just utilized and unmodified version of the title.

There may be some cleverer ways of performing comparisons if the naming patterns could be confirmed, but I don't think any associated performance benefits are even worth investigating given the almost certainly minimal gains on this which is a one-time flow on the startup path